Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/MaterialComponentController.cpp

```diff
@@ -130,6 +131,7 @@ namespace AZ
 
         void MaterialComponentController::Deactivate()
         {
+            AZ::Data::AssetBus::MultiHandler::BusDisconnect();
             MaterialComponentRequestBus::Handler::BusDisconnect();
             MaterialConsumerNotificationBus::Handler::BusDisconnect();
 
```

Comment:
What happened to the whitespace here?  Sometimes its just a github glitch of course.

Reply:
Maybe a glitch? I don't see any extra whitespace.